lib/graphql/tracing/platform_tracing.rb

```diff
@@ -102,6 +102,21 @@ def self.use(schema_defn, options = {})
         schema_defn.tracer(tracer)
       end
 
+      protected
```

Comment:
Just curious, why `protected` instead of `private`? (I have yet to find a case where I really _want_ `protected`, maybe today's the day 😆 !)

Reply:
I went ahead and switched it to private. In general, I prefer the _most_ privacy I can get, so that refactoring is easier and less risky, but if I've misunderstood, please let me know!